### PR TITLE
AST-1456 - Jerk issue while switching to dark mode site

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- First it loads color palette 1 then switching to dark palette there is jerk.

### Screenshots
- https://share.bsf.io/GGu49mYg

### Types of changes
- Bug fix 

### How has this been tested?
- Enable dark OS setting
- Choose OS Aware setting for first load
- Then check in incognito mode, jerk will be there

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards
- [x] My code has proper inline documentation
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/class-astra-builder-base-dynamic-css.php [hunk 1]
```diff
@@ -560,14 +560,22 @@ public static function dark_color_palette_css( $dynamic_css, $dynamic_css_filter
...
+						carryOsSetting = "<?php echo astra_get_option( 'mode-switcher-carry-os-palette', true ); ?>";
+					if ( siteView && '' !== siteView ) {
+						if ( 'dark' === siteView && ! document.documentElement.classList.contains( 'ast-dark-mode' ) ) {
+							document.documentElement.classList.add( 'ast-dark-mode' );
+						} else if ( 'light' === siteView && document.documentElement.classList.contains( 'ast-dark-mode' ) ) {
+							document.documentElement.classList.remove( 'ast-dark-mode' );
+						}
+					} else if( '1' === carryOsSetting ) {
+						// Logic for OS Aware option to showcase site on load with their set system scheme.
+						var hasDarkSchemeSupport = window.matchMedia( "(prefers-color-scheme: dark)" );
```
Comment (GauravKhupse) [inline inc/builder/type/class-astra-builder-base-dynamic-css.php:573]: @imnavanath Can we add limited scope **let** for this?

Comment (imnavanath) [reply inc/builder/type/class-astra-builder-base-dynamic-css.php]: Done 👍

Comment (premanshup) [reply inc/builder/type/class-astra-builder-base-dynamic-css.php]: @imnavanath I guess some browsers do not support let keywords including IE 11. Please check.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#browser_compatibility
cc: @GauravKhupse

**Review by premanshup — APPROVED** [review]

File: inc/builder/type/class-astra-builder-base-dynamic-css.php (hunk 1 above)
Comment (imnavanath) [reply inc/builder/type/class-astra-builder-base-dynamic-css.php]: True, 👍 reverted let to var.

**Pull request merged**